docs/source/user_guide/micromamba.rst

```diff
@@ -32,7 +32,7 @@ Quickstarts
     config                      Configuration of micromamba
     info                        Information about micromamba
     constructor                 Commands to support using micromamba in constructor
-    env                         List environments
+    env                         Access information about environments
```

Comment:
I think this should be pointing to `env` command help (like `conda`).
```suggestion
    env                         See `mamba/micromamba env --help`.
```
Same for everything below.

Reply:
What would you prefer as a user, @corneliusroemer?

Reply:
Small up @corneliusroemer.